### PR TITLE
Refactor/fullspace bugfix

### PR DESCRIPTION
**Pull request by orianac**
@norlandrhagen - would be great to get a review on this! thanks!! 🐈 🐈‍⬛ ❤️

### CODE REVIEW

File: cmip6_downscaling/methods/bcsd/tasks.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ def fit_and_predict(
     coarse_obs_full_time_ds = xr.open_zarr(coarse_obs_full_time_path)
     experiment_train_full_time_ds = xr.open_zarr(experiment_train_full_time_path)
     experiment_predict_full_time_ds = xr.open_zarr(experiment_predict_full_time_path)
-
+    experiment_train_full_time_ds['time'] = coarse_obs_full_time_ds.time.values
```
Comment (norlandrhagen) [inline cmip6_downscaling/methods/bcsd/tasks.py:136]: Maybe we add a short note explaining why this is needed incase we see a similar error in the future.

File: cmip6_downscaling/methods/bcsd/tasks.py [hunk 2]
```diff
@@ -171,7 +171,7 @@ def postprocess_bcsd(
         UPath to post-processed dataset.
     """
 
-    target = results_dir / "interpolate_prediction" / run_parameters.run_id
+    target = results_dir / "daily" / run_parameters.run_id
```
Comment (norlandrhagen) [inline cmip6_downscaling/methods/bcsd/tasks.py:174]: maybe "post-process" instead of daily?

**Review by norlandrhagen — COMMENTED** [review]
Looks great! Just a note for future us -- at this point we were able to successfully run a full-space single year run from start to finish on a jupyter-lab instance.

**Comment by orianac** [comment]
@jhamman @norlandrhagen 
As of commit [3fca939](https://github.com/carbonplan/cmip6-downscaling/pull/144/commits/3fca939ced6a50d888e1b17741b8c01eccf0486f) both a full space (1 year train, 1 year predict) and a small space (10x10 degree, 30 year train, 130 year predict) works with this setup:
- pip installing the upath fix (in the latest envs that @jhamman is installing) 
- change to kernel name in postbuild file housed in the same image fix as is getting handled in the bullet above
- rechunker v. 0.4.2 (this is NOT currently the image - we're currently using 0.3.3 - @norlandrhagen could we update that?)
currently trying out a full run!

File: cmip6_downscaling/methods/bcsd/tasks.py (hunk 2 above)
Comment (orianac) [reply cmip6_downscaling/methods/bcsd/tasks.py:174]: i think that marking it `daily` denotes that it is a final output file whereas `postprocess` doesn't really mean much to me at least? the most important flag seems that it is in the `results_dir`, and from there we distinguish between timesteps. it also plays nice with the monthly/summary files that are named `monthly_summary` and `annual_summary`. but i'd be open to changing the `monthly_summary`

File: cmip6_downscaling/methods/bcsd/tasks.py (hunk 1 above)
Comment (orianac) [reply cmip6_downscaling/methods/bcsd/tasks.py:136]: great point! i'll add a comment.

Comment (orianac) [reply cmip6_downscaling/methods/bcsd/tasks.py:138]: added a comment - @norlandrhagen - any chance you'd like to tackle this TODO?

**Pull request merged**